.github/workflows/pr.yml

```diff
@@ -16,23 +16,23 @@ jobs:
         with:
           node-version: "${{ matrix.node }}"
           cache: yarn
-      - name: Setup Anchor & Build
-        uses: metadaoproject/setup-anchor@v2
```

Comment:
this used `@coral-xyz/anchor-cli` npm package which has not been updated for a while, hence switching to `solana-developers/github-actions` which autodetects the used versions and installs them.